astropy/utils/data.py

```diff
@@ -1854,7 +1854,7 @@ def clear_download_cache(hashorurl=None, pkgname="astropy"):
                 filepath = os.path.join(dldir, d)
             if os.path.exists(filepath):
                 _rmtree(filepath)
-            elif len(hashorurl) == 2 * hashlib.md5().digest_size and re.match(
+            elif len(hashorurl) == 2 * hashlib.md5(usedforsecurity=False).digest_size and re.match(
```

Comment:
This line is pretty long. I bet that's what the formatted is unhappy about. We shouldn't need to rebase, running pre-commit.ci autofix should suffice.

Reply:
Hm, running pre-commi.ci autofix didn't seem to do it and neither did breaking up the line manually. 1f98c48